### DISCUSSION TITLE
Short Namespaces on mobile?

### DISCUSSION THREAD

**Discussion by abhilesh** [YU000jp/logseq-plugin-short-namespaces#28] [Q&A]
I use namespaces extensively to manage my projects and end up with namespaces that are several levels deep. This plugin is immensely helpful in managing the deep hierarchies I have setup in my graph. I would also like to have this functionality on the mobile Logseq client.

Is it possible to add some of the code to the `custom.js` file to replicate this functionality on mobile? If so, could you please point me to the appropriate place?

**Comment by YU000jp** [comment]
https://discord.com/channels/725182569297215569/896368413243494430/1116515366508711996
There is a way without using this plugin. However, I have not verified that the script works on mobile. Mobile operation may become slow.
